istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -167,9 +204,70 @@ spec:
       - name: istio-certs
         secret:
           secretName: istio.istio-pilot-service-account
-          optional: false
+          optional: true
```

Comment:
Intentionally changed - pilot should not be scheduled if the certs are missing, subtle problems.